frontend/src/Screens/GlobalComponents/Navbar/Components/PageBoard/PageBoard.tsx

```diff
@@ -1,11 +1,34 @@
 import './PageBoard.css';
+import { useEffect } from "react";
+
+import myApi from "../../../../../Url/Url";
 
 function PageBoard() {
+    useEffect(() => {
+        getTasksByIdProject("blabla")
+    }, []); 
+
+    const getTasksByIdProject =async (idProject:string) => {
+        try {
+            const result = await Promise.all([
```

Comment:
please use an [distracturing] fot the results

Reply:
I was change it in the future branch